website/login.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+	<head lang="en"><style type="text/css">@charset "UTF-8";[ng\:cloak],[ng-cloak],[data-ng-cloak],[x-ng-cloak],.ng-cloak,.x-ng-cloak,.ng-hide:not(.ng-hide-animate){display:none !important;}ng\:form{display:block;}.ng-animate-shim{visibility:hidden;}.ng-anchor{position:absolute;}</style>		
```

Comment:
I'm a little confused why you have Angular styling (as we aren't using Angular for this project). Could you explain what it's doing?

Reply:
I saw another login page I liked online, where there was only 1 form to enter email and password, and the user can either register with that information or login to an existing account. I went through it to figure out how they made it work and realized they used angular but at that point I had already been working at it for a while and I signed up to work on this 10 days ago so I just wanted to submit something. Since we're not using a password field at all, it can just be deleted.

Reply:
I figured it was something like that, but just wanted to make sure. Playing with example code is a great way to learn how things work